assets/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+ul {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+}
+
+body {
+  background-color: #f7f7f9;
+  position: relative;
+}
+
+main {
+  background-image: url("../img/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-color: #fff;
+  border-bottom-left-radius: 80px;
+}
+
+header {
+  position: fixed;
+  background-color: #fff;
+  top: 0;
+  width: 100%;
+  padding: 1rem 0;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+header a {
+  margin-left: 24px;
+  color: #6070ff;
+  font-family: "Poppins", sans-serif;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+}
+
+.fa-bars {
+  margin-right: 24px;
+  color: #6070ff;
+}
+
+.section-top {
```

Comment:
- [ ] Please make sure that you use CSS Flexbox to style this section as well. (The same as you did in the `<header>` section).
It's important that you follow the project requirements for best practices. You're almost there 👍

Reply:
When doing flexbox on CSS is not necessary to make all the areas have it, is of good practice to know that, I have been in Microverse since 2019-2020, made 4 months of HTML and CSS, and quit before my capstone project due illness, and I just got back, so I also know this part. Plus by adding flex to that part, it just messes up the whole structure of the project, which already looks good, im using a live version on vs code and it looks good, even the icons so idk.

Reply:
I suggest you implement the changes requested by the previous reviewer its part of the project requirement 🙏 
project requirements are like real-life project specification given to you by your future employers that's why we should adhere to them 👍

Reply:
I suppose you didn't check my code updates, I DID implement his suggestion and it broke my code, it doesn't work, and it can't be okay if his suggestion breaks the code. I think we should move on from this, plus as long as I added flex box on the CSS is not needed for it to be in each line of CSS code.

Reply:
I added it the way he said and it broke the code, so umm Ima try another option with flex then.

Reply:
ok fixed it.